src/Logging/InitializationLogger.cs

```diff
@@ -41,7 +41,7 @@ private static ILoggingBuilder LoadInitializationLogger(this ILoggingBuilder bui
 		public static void LogInitializationSucceed(string serviceNameForLogging, string message = "")
 		{
 			string logMessage = $"Initialization successful for {serviceNameForLogging}, {message}";
-			ServiceInitializationEventSource.Instance.LogHostBuildSucceeded(Process.GetCurrentProcess().Id, serviceNameForLogging, logMessage);
+			ServiceInitializationEventSource.Instance.LogHostBuildSucceeded(Environment.ProcessId, serviceNameForLogging, logMessage);
```

Comment:
Environment.ProcessId doesn't exist in NetStandard 2.0

Reply:
@artempushkin - is there any reason the Microsoft.Omex.Extensions.Logging should support NetStandard 2.0?
I'd assume at this stage we wouldn't want to support FullFramework in downstream consumers, right?

Reply:
We do target netframework in a few services, mostly in OmexAddInCertification. I suppose there can be partner dependencies forcing full .net.

Reply:
I went ahead with using conditional compilation, but I feel like we should move away from net standard in future.